renku/cli/init.py

```diff
@@ -95,97 +99,185 @@ def store_directory(ctx, param, value):
     return value
 
 
-def template(client, force):
-    """Render templated configuration files."""
-    import pkg_resources
+def create_template_sentence(templates):
+    """Create templates choice sentence.
 
-    # create the templated files
-    for tpl_file in CI_TEMPLATES:
-        tpl_path = client.path / tpl_file
-        with pkg_resources.resource_stream(__name__, tpl_file) as tpl:
-            content = tpl.read()
+    :ref templates: list of templates coming from manifest file
+    """
+    template_sentences = []
+    for index, template_elem in enumerate(templates):
+        template_sentences.append(
+            '[{0}] {1}'.format(index + 1, template_elem['name'])
+        )
+    template_sentence = ', '.join(template_sentences)
+    return (
+        'Please choose a template by typing the number '
+        '({0}) or [0] to print the description'.format(template_sentence)
+    )
 
-            if not force and tpl_path.exists():
-                click.confirm(
-                    'Do you want to override "{tpl_file}"'.format(
-                        tpl_file=tpl_file
-                    ),
-                    abort=True,
-                )
 
-            with tpl_path.open('wb') as dest:
-                dest.write(content)
+def create_printable_descriptions(templates):
+    """Create description string of the templates.
+
+    :ref templates: list of templates coming from manifest file
+    """
+    template_descriptions = []
+    for index, template in enumerate(templates):
+        template_descriptions.append(
+            '[{0}] {1}: {2}'.format(
+                index + 1, template['name'], template['description']
+            )
+        )
+    return '\n'.join(template_descriptions)
 
 
 @click.command()
 @click.argument(
-    'directory',
+    'path',
     default='.',
     callback=store_directory,
-    type=click.Path(writable=True, file_okay=False, resolve_path=True)
+    type=click.Path(writable=True, file_okay=False, resolve_path=True),
+)
+@click.option(
+    '--name',
```

Comment:
Is this option needed? If `init` is called with an argument, e.g. `renku init my-renku-project` the name is `my-renku-project`. If it is called within an existing directory the name is the directory name. I suppose you could still give it a name explicitly, but the default _should not_ be "Renku project".  I didn't notice that this had been set before, I just realized when I was taking a closer look at the triples that are generated...

Reply:
right, I restored the previous functionality since I would keep the possibility to give a custom name to the project but falling back to the folder name when this is not provided